container/start.sh

```diff
@@ -40,10 +40,11 @@ start_karapace_registry(){
     "compatibility": "FULL",
     "log_level": "${KARAPACE_REGISTRY_LOG_LEVEL:-$KARAPACE_REGISTRY_LOG_LEVEL_DEFAULT}",
     "replication_factor": 1,
-    "security_protocol": "PLAINTEXT",
-    "ssl_cafile": null,
-    "ssl_certfile": null,
-    "ssl_keyfile": null
+    "security_protocol": "${KARAPACE_SECURITY_PROTOCOL:-PLAINTEXT}",
+    "ssl_cafile": "${KARAPACE_SSL_CA_FILE_PATH:-null}",
+    "ssl_certfile": "${KARAPACE_SSL_CERT_FILE_PATH:-null}",,
+    "ssl_keyfile": "${KARAPACE_SSL_KEY_FILE_PATH:-null}",
+    "ssl_check_hostname": "${KARAPACE_SSL_CHECK_HOSTANME:-True}"
```

Comment:
This produces invalid JSON, because it has quotes around `null` and booleans should be lowercased ie. `true` instead of `True` inside JSON file. Also there is double `,,` after certfile and HOSTANME spelling mistake
```
    "ssl_keyfile": "null",
    "ssl_check_hostname": "True"
```

Reply:
Resolved the issues, 10x